contributor_path.rst

```diff
@@ -0,0 +1,212 @@
+****************
+Contributor path
+****************
+
+..
+
+  "There is no favorable wind for the sailor who doesn’t know where to go."
+
+  -- Seneca
+
+This guide describes a contributor path in scope of the Ansible project from the early beginning, through all the stages, to becoming a leader shaping the project's future and the future of automation in the IT world in general.
+
+This path is not a policy but rather a suite of recommendations based on experience of successful contributors.
+
+The goal of the path, first of all, is to make your activity more determined and efficient through understanding of what you can achieve and how. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. When you contribute regularly, your proficiency and judgement in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+Regular contribution makes you a member of the community and leads to respect of other community members. Do you respect people who give good to others? Do you respect people who help you as a user whose feature request was implemented or who help you as a programmer to finish your pull request?
+
+Be persistent and act gradually. Great result is just a product of time and effort. Let's go then!
```

Comment:
This feels slightly weird - you start with "take it slow" and end with "lets go!" :)

Perhaps "lets go then" could be "Here's how to get started" or something like that?

(This is a nitpick / style issue, you can disagree, of course :P)

Reply:
I like your wording, thanks for that! Changed